binr/radiff2/radiff2.c

```diff
@@ -34,6 +34,7 @@ static RCore* opencore(const char *f) {
 	const ut64 baddr = UT64_MAX;
 	RCore *c = r_core_new ();
 	r_core_loadlibs (c, R_CORE_LOADLIBS_ALL, NULL);
+	if (!c) return NULL;
```

Comment:
this check probably needs to go one line before. just after `RCore *c = r_core_new ();`